zebrad/tests/acceptance.rs

```diff
@@ -1706,6 +1706,57 @@ async fn delete_old_databases() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn stored_config_works() -> Result<()> {
```

Comment:
It will be easier to call this function if it's not async.
(I don't think it needs to be async.)